package.json

```diff
@@ -183,6 +196,7 @@
     "immer": "^10.1.1",
     "immutable": "^5.1.2",
     "io-ts": "^2.2.22",
+    "jsdom": "^26.1.0",
```

Comment:
Why is `jsdom` not a dev dependency?

Reply:
Same question, why not a dev dep? also, we probably don't really need as we can use the defaultvitest environment "node". Mol* runs in both, the browser and node. Any particular reason to use jsdom here?